### PR TITLE
Add automatic minor increase on feature

### PR DESCRIPTION
**Pull request by bidoubiwa**
When a new feature is added to a project that already has a fist major version, the update of the semver goes the following way: 

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

This does not apply on packages with no major version. 

To avoid having to manually update the minor version on new features, this PR adds the automation of the minor increase when using the `new-feature` label. 

### feature 👍

The name `feature` is based on the following common practices: 
[GitHub Labels that are logical, colorful and sensible](https://docs.saltproject.io/en/latest/topics/development/labels.html#labels)
[GitLab suggested labels](https://docs.gitlab.com/ee/user/project/labels.html)
> Categorize epics, issues, and merge requests using colors and descriptive titles like bug, feature request, or docs.

Take into account that these are two suggested naming. A lot of other articles exists with different suggestion and the first article also invites to use `type:` in front of the label.

### new-feature 🎉
Another possibility would be to use `new-feature` as it is more explicit linked to a PR and not an issue.

I suggest `new-feature` instead of `new feature` for consistency with `breaking-changes` and `skip-changelog`

If you prefer `feature` 👍
If you prefer `new-feature` 🎉

⚠️ Wait for [this pr](https://github.com/meilisearch/integration-guides/pull/129) before merging

### CODE REVIEW

File: .github/release-draft-template.yml [hunk 1]
```diff
@@ -6,10 +6,15 @@ version-resolver:
   major:
     labels:
       - 'breaking-change'
+  minor:
+    labels:
+      - 'feature'
```
Comment (curquiza) [inline .github/release-draft-template.yml:11]:
```suggestion
      - 'new-feature'
```

File: .github/release-draft-template.yml [hunk 2]
```diff
@@ -6,10 +6,15 @@ version-resolver:
...
   default: patch
 categories:
   - title: 'Breaking changes ⚠️'
     label: 'breaking-change'
+  - title: 'New features ⚡️'
+    label: 'feature'
```
Comment (curquiza) [inline .github/release-draft-template.yml:17]:
```suggestion
    label: 'new-feature'
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -122,6 +122,10 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
 This project integrates a tool to create automated changelogs.<br>
 _[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md)._
 
+### Labels
+
+To chose the proper labels reade the [labels guide](https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md#how-does-the-release-drafter-work)). Labels impact the generated version for the next release.
+
```
Comment (curquiza) [inline CONTRIBUTING.md:128]: There is already the explanation in the guide, no need to make the CONTRIBUTING longer. Plus, the `label` title is out of context for someone who doesn't know the release drafter -> they don't know why we are talking about labels.
I would rather keep all the information of the release drafter in https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md 🙂

**Review by curquiza — CHANGES_REQUESTED** [review]

**Review by curquiza — APPROVED** [review]
Awesome! thanks!
I've just added the `new-feature` label to this repository! :)

**Review by mdubus — APPROVED** [review]

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**